### DISCUSSION TITLE
Samples to share with the community?

### DISCUSSION THREAD

**Discussion by JobadSayas** [pybricks/support#960] [Ideas]
I was wondering if there is somewhere for the community to share samples, it would be useful to see others code to learn and also cool for one to share own creations with the others.

Maybe that place already exist so I would like to know which, if not, then please consider it.

**Comment by dlech** [comment]
Here on the discussion forum under the Show and Tell tag is a good place. If you have something more polished that is easy for others to build and want to share it at https://pybricks.com/projects/, you can make a pull request to https://github.com/pybricks/pybricks-projects.
